tap_github/client.py

```diff
@@ -195,32 +208,37 @@ def authed_get(self, source, url, headers={}, stream="", should_skip_404 = True)
         """
         Call rest API and return the response in case of status code 200.
         """
-        with metrics.http_request_timer(source) as timer:
+        with metrics.http_request_timer(url) as timer:
             self.session.headers.update(headers)
             resp = self.session.request(method='get', url=url, timeout=self.get_request_timeout())
             if resp.status_code != 200:
                 raise_for_error(resp, source, stream, self, should_skip_404)
             timer.tags[metrics.Tag.http_status_code] = resp.status_code
             rate_throttling(resp, self.max_sleep_seconds, self.min_remain_rate_limit)
-            if resp.status_code == 404:
+            if resp.status_code == 404 or resp.status_code == 422:
```

Comment:
No need to check the response message again because if the error code is 422 and it's not a pagination error, the `raise_for_error` method will raise an exception.